netbox/ipam/models.py

```diff
@@ -559,14 +559,14 @@ def get_utilization(self):
                 vrf=self.vrf
             )
             child_prefixes = netaddr.IPSet([p.prefix for p in queryset])
-            return int(float(child_prefixes.size) / self.prefix.size * 100)
+            return (int(float(child_prefixes.size) / self.prefix.size * 100), child_prefixes.size, self.prefix.size)
```

Comment:
Returning the numerator, denominator, and quotient is redundant: We can return just the child prefixes' size and parent prefix size, and let the caller calculate the percentage.

Also, since we're changing the return signature, be sure to update the method's docstring.